### PR TITLE
introducing 2 new workshops and update in 1 existing workshop (WMS 10001) & (WMS 10002)

### PR DESCRIPTION
**Pull request by romitacharya**
2 new workshops:

- EM patching using Fleet UI (WMS 10001)
- EM Upgrade using Fleet UI (WMS 10002)

Update in existing workshop

- EM upgrade (this change will make both command line and UI based labs in sync, with no drifters) (WMS 1680)

### CODE REVIEW

**Comment by klazarz** [comment]
Hi @romitacharya, could you please add the WMS ID(s). Thanks, Keivn

**Comment by romitacharya** [comment]
> 

Hi @klazarz , I have added it. Thanks, Romit

File: enterprise-manageability-library/enterprise-manager/workshops/livelabs/manifest.json [hunk 1]
```diff
@@ -37,11 +37,23 @@
       "filename": "./../../em-fleet-maintenance-patching/em-fleet-maintenance-patching.md"
     },
     {
+      "title": "Lab 5A: Automated Database Patching at Scale with Fleet Maintenance UI",
```
Comment (rfontcha) [inline enterprise-manageability-library/enterprise-manager/workshops/livelabs/manifest.json:40]: Lab # are all digits. Please use sequences and not 6A/7A, etc.

**Review by rfontcha — CHANGES_REQUESTED** [review]
@romitacharya . Please use this for :
- "Lab 2: Environment Setup" in all Freetier manifest.json: https://raw.githubusercontent.com/oracle/learning-library/master/common/labs/setup-compute-generic/setup-compute-novnc-ssh.md
- "Lab 1: Environment Setup" in all "livelabs manifest.json: https://raw.githubusercontent.com/oracle/learning-library/master/common/labs/verify-compute/verify-compute-ssh-and-novnc.md

Also modify all Labs numbers to sequences. not 6A, 7A etc. Must be a number

File: enterprise-manageability-library/enterprise-manager/workshops/livelabs/manifest.json (hunk 1 above)
Comment (romitacharya) [reply enterprise-manageability-library/enterprise-manager/workshops/livelabs/manifest.json]: Thanks @rfontcha . I have made the necessary changes. UI workshop is not part of fundamental workshop for now, and I have removed the references from manifest.json file.

**Comment by romitacharya** [comment]
> @romitacharya . Please use this for :
> 
> * "Lab 2: Environment Setup" in all Freetier manifest.json: https://raw.githubusercontent.com/oracle/learning-library/master/common/labs/setup-compute-generic/setup-compute-novnc-ssh.md
> * "Lab 1: Environment Setup" in all "livelabs manifest.json: https://raw.githubusercontent.com/oracle/learning-library/master/common/labs/verify-compute/verify-compute-ssh-and-novnc.md
> 
> Also modify all Labs numbers to sequences. not 6A, 7A etc. Must be a number

Hi @rfontcha Suggested changes done..

File: enterprise-manageability-library/enterprise-manager/workshops/livelabs/manifest.json (hunk 1 above)
Comment (rfontcha) [reply enterprise-manageability-library/enterprise-manager/workshops/livelabs/manifest.json]: @romitacharya  - Remove all "../workshop/main-*". They're no longer used. Also make sure that your repo is in sync by running "Merge into current branch" and selecting "upstream/master"

**Review by rfontcha — APPROVED** [review]

**Pull request merged**